packages/analytics/amplify_analytics_pinpoint/example/backend/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "backend",
+  "version": "0.1.0",
+  "bin": {
+    "backend": "bin/backend.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk",
+    "deploy": "cdk deploy --outputs-file=outputs.json && tool/create_config.dart"
```

Comment:
I got an error first time I ran this bc needed to run `cdk boostrap`. Can that be added here?

Reply:
Maybe I can add it to the README? `cdk bootstrap` only needs to be run once per account in order to use CDK in general. It's not project-specific, though, and I don't think calling it multiple times is best.

Reply:
Actually, I take that back. Looks like bootstrap can, and maybe even should, be run multiple times, since the resources it deploys need to be updated occasionally. I've added a separate npm script to do this though so it's only called when running the provisioning script or manually.